docs/docs/version-comparison.md

```diff
@@ -2,59 +2,126 @@
-- The SaaS and Enterprise versions have [Role-Based Access Control](/system-administration/rbac).
-- The SaaS and Enterprise versions have [Audit Logs](/system-administration/audit-logs).
-- The SaaS and Enterprise versions have additional Authentication Providers:
-  - [Okta](/system-administration/authentication/Okta)
-  - [LDAP](/system-administration/authentication/LDAP)
-  - [SAML](/system-administration/authentication/SAML)
-  - [ADFS](/system-administration/authentication/ADFS)
-
-:::tip
-
-You can switch between SaaS and Self Hosted Flagsmith using our
-[Import and Export tools](system-administration/importing-and-exporting/organisations).
+import Card from '@site/src/components/Card';
+import CardHeader from '@site/src/components/Card/CardHeader';
+import CardBody from '@site/src/components/Card/CardBody';
+import CardFooter from '@site/src/components/Card/CardFooter';
 
-:::
-
-## SaaS Benefits
+# Version Comparison
 
-Our SaaS platform has a number of benefits:
+Choose the right Flagsmith version for your needs. Compare our Open Source, SaaS, and Enterprise offerings:
 
-- You can get up and running right away.
-- Our global [Edge API](advanced-use/edge-api.md) provides global low latency flags. We aim to serve all flag requests
-  in < 150ms, globally.
-- Get real-time flag updates to your clients, the moment they are changed in the dashboard.
-- We deal with platform upgrades, security patches, scaling and backups.
+## Feature Comparison
 
-## Enterprise Benefits
+|  | Free | Saas | Enterprise |
+|---------|:-------------:|:------:|:------------:|
+| **Core Features** |
+| Feature Flags | ✅ | ✅ | ✅ |
+| Remote Config | ✅ | ✅ | ✅ |
+| User Segments | ✅ | ✅ | ✅ |
+| A/B Testing | ✅ | ✅ | ✅ |
+| 3rd Party Integrations | ✅ | ✅ | ✅ |
+| **Scaling** |
+| API Request Limits | 50k/month | 1M/month | 5M+/month |
+| Team Members | 1 | 3 | 20+ |
+| Projects | Single | Unlimited | Unlimited |
+| Environments | Unlimited | Unlimited | Unlimited |
+| **Advanced Features** |
+| Change Requests | ❌ | ✅ | ✅ |
+| Flag Scheduling | ❌ | ✅ | ✅ |
+| Role-Based Access Control | ❌ | ✅ | ✅ |
+| Audit Logs | ❌ | ✅ | ✅ |
+| Two-Factor Authentication | ❌ | ✅ | ✅ |
+| Priority Support | ❌ | Email | Slack/Discord |
+| **Enterprise Features** |
+| Custom SLA | ❌ | ❌ | ✅ |
+| On-Boarding and Training | ❌ | ❌ | ✅ |
+| Custom Fields | ❌ | ❌ | ✅ |
+| **Authentication** |
+| Email/Password | ✅ | ✅ | ✅ |
+| OAuth | ✅ | ✅ | ✅ |
+| SAML/SSO | ❌ | ❌ | ✅ |
+| LDAP | ❌ | ❌ | ✅ |
+| Okta | ❌ | ❌ | ✅ |
+| **Infrastructure** |
+| Self-hosted | ✅ | ❌ | Optional |
+| Cloud Hosted | ❌ | ✅ | Optional |
+| Private Cloud | ❌ | ❌ | ✅ |
+| Edge API | ❌ | ✅ | Optional |
+| Database Options | PostgreSQL | PostgreSQL | PostgreSQL, Oracle, MySQL |
 
-You can run our Enterprise version either on-premise, or we can provide private cloud instance dedicated to your
-organisation.
+## Deployment Options
```

Comment:
What's being described here are SaaS plans, not deployment options (same as https://www.flagsmith.com/pricing). I would also prefer to avoid mentioning any specific pricing plans and limits in the documentation.